skimage/measure/_find_contours.py

```diff
@@ -30,6 +31,11 @@ def find_contours(array, level,
          contours will wind counter- clockwise around elements below the
          iso-value. Alternately, this means that low-valued elements are always
          on the left of the contour. (See below for details.)
+    nodata : float
```

Comment:
I don't really find this name descriptive. What do you think about `ignore_value` or `ignore_as_missing`?

Reply:
I followed the suggestion of `nodata` from #3006, but I'm not particularly attached to it - something like  `ignore_as_missing` or `missing_data_value` would indeed be more descriptive.